File: Atcoder/Ruby/236.ABC/B.rb
mori = gets.to_i
yama = gets.split.map(&:to_i)


kes = 0
yama.each do |i|
  kes += i
end

fuji = 0
(1..mori).each do |i|
  fuji += i*4
end

puts fuji-kes

File: Atcoder/PHP/236.ABC/B.php
<?php
fscanf(STDIN, "%d", $kes);
$mori = explode(' ', trim(fgets(STDIN)));
$yama = array_count_values($mori);

foreach ($yama as $fuji => $taku) {
    if ($taku != 4) {
        echo $fuji;
    }
}
?>